llvm/test/Analysis/Lint/const-store.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: not opt --mtriple=amdgcn --passes=lint --lint-abort-on-error %s -o /dev/null |& FileCheck %s
+; RUN: opt --mtriple=amdgcn --mcpu=gfx1030 --passes=lint %s -o /dev/null |& FileCheck %s --check-prefixes=CHECK,CHECK0
+; RUN: opt --mtriple=x86_64 --passes=lint --lint-abort-on-error %s -o /dev/null |& FileCheck %s --allow-empty --check-prefix=NOERR
+; NOERR: {{^$}}
+
```

Comment:
I think you will need `; REQUIRES: amdgpu-registered-target`.

Reply:
This test doesn't actually require the targets. The triple will still be recognized no matter what, which is all that's needed